File: 0001-two-sum/0001-two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> result;

        for (int i = 0; i < nums.size(); i++) {     // outer loop
            for (int j = i+1; j < nums.size(); j++) { // inner loop
                // iteration

                int sum = nums.at(i) + nums.at(j);
                if (sum == target) {
                    result = {i, j};
                    break;
                }
            }
        }

        return result;
    }
};

File: 0001-two-sum/0001-two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        result = []

        for i in range(0, len(nums)): #outer loop
            for j in range(i+1, len(nums)): #inner loop
                #iteration

                sum = nums[i] + nums[j]
                if sum == target:
                    result.insert(0, i)
                    result.insert(1, j)
                    break
        
        return result

            



        